app/src/main/java/com/github/libretube/ui/sheets/StatsSheet.kt

```diff
@@ -33,7 +36,12 @@ class StatsSheet : ExpandedBottomSheet() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         val binding = binding
 
+        val clipboard = context?.let { getSystemService(it, ClipboardManager::class.java) }
```

Comment:
```suggestion
        val clipboardManager = context?.getSystemService(ClipboardManager::class.java)!!
```
The `context` can't be null here since we're in `onViewCreated` (at the beginning of the lifecycle), so it can't yet be destroyed.

Reply:
Yeah, I kind of suspected that, but I wasn't quite sure, so I figured better safe than sorry :)